src/docs/go-grpc/sections/02-try/02-server.mdx

```diff
@@ -0,0 +1,78 @@
+We're going to create a simple gRPC server in Go that loads an X.509 certificate
+and its key to authenticate to clients using just server authentication.
```

Comment:
```suggestion
We're going to create a simple gRPC server in Go that loads an X.509 certificate and its key,
so that clients can connect using TLS server authentication.
```